datasets/census_bureau_acs/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -0,0 +1,550 @@
+        "B08014002": "no_car",
+        "C24060004": "sales_office_employed",
+        "C24060002": "management_business_sci_arts_employed",
+        "B23006001": "pop_25_64",
+        "B23006023": "bachelors_degree_or_higher_25_64",
+        "B11001007": "nonfamily_households",
+        "B11001002": "family_households",
+        "B25035001": "median_year_structure_built",
+    }
+
+    state_code = {
+        "01": "Alabama",
+        "02": "Alaska",
+        "04": "Arizona",
+        "05": "Arkansas",
+        "06": "California",
+        "08": "Colorado",
+        "09": "Connecticut",
+        "10": "Delaware",
+        "11": "District of Columbia",
+        "12": "Florida",
+        "13": "Georgia",
+        "15": "Hawaii",
+        "16": "Idaho",
+        "17": "Illinois",
+        "18": "Indiana",
+        "19": "Iowa",
+        "20": "Kansas",
+        "21": "Kentucky",
+        "22": "Louisiana",
+        "23": "Maine",
+        "24": "Maryland",
+        "25": "Massachusetts",
+        "26": "Michigan",
+        "27": "Minnesota",
+        "28": "Mississippi",
+        "29": "Missouri",
+        "30": "Montana",
+        "31": "Nebraska",
+        "32": "Nevada",
+        "33": "New Hampshire",
+        "34": "New Jersey",
+        "35": "New Mexico",
+        "36": "New York",
+        "37": "North Carolina",
+        "38": "North Dakota",
+        "39": "Ohio",
+        "40": "Oklahoma",
+        "41": "Oregon",
+        "42": "Pennsylvania",
+        "44": "Rhode Island",
+        "45": "South Carolina",
+        "46": "South Dakota",
+        "47": "Tennessee",
+        "48": "Texas",
+        "49": "Utah",
+        "50": "Vermont",
+        "51": "Virginia",
+        "53": "Washington",
+        "54": "West Virginia",
+        "55": "Wisconsin",
+        "56": "Wyoming",
+        "60": "American Samoa",
+        "66": "Guam",
+        "69": "Commonwealth of the Northern Mariana Islands",
+        "72": "Puerto Rico",
+        "78": "United States Virgin Islands",
+    }
+
+    logging.info("Extracting the data from API and loading into dataframe...")
+    if report_level == "national_level":
+        df = extract_data_and_convert_to_df_national_level(
+            group_id, year_report, api_naming_convention, source_url
+        )
+    elif report_level == "state_level":
+        df = extract_data_and_convert_to_df_state_level(
+            group_id, state_code, year_report, api_naming_convention, source_url
+        )
+
+    logging.info("Replacing values...")
+    df = df.replace(to_replace={"KPI_Name": group_id})
+
+    logging.info("Renaming headers...")
+    rename_headers(df, rename_mappings)
+
+    logging.info("Creating column geo_id...")
+    if geography == "censustract" or geography == "blockgroup":
+        df["tract"] = df["tract"].apply(change_length, args=("6"))
+        df["state"] = df["state"].apply(change_length, args=("2"))
+        df["county"] = df["county"].apply(change_length, args=("3"))
```

Comment:
```suggestion
        df["tract"] = df["tract"].apply(change_length, args=(6))
        df["state"] = df["state"].apply(change_length, args=(2))
        df["county"] = df["county"].apply(change_length, args=(3))
```
and perhaps use the function name `pad_zeroes_to_the_right` or something in that regard, to make it more descriptive

Reply:
done